surveys/2024-annual-survey/translations/es.md

```diff
@@ -579,8 +579,8 @@ Type: matrix
 
 Rows:
 
-- Es fácil para los solicitantes cualificados encontrar trabajos que usan Rust para la mayoría de la programación
-- 
+- Es fácil para los solicitantes cualificados encontrar trabajos que usan Rust para la mayor parte de la programación
+- Es sencillo, para programadores mayores, encontrar empleos en los cuales Rust es utilizado en la mayor parte de la programación.
```

Comment:
programadores or developers? :-) Makes sense using always the same word?

Reply:
We were using developers everywhere except here, but I changed it now to use programadores.
If we already offer translations, we might as well use spanish words ^^